Core/RetroAchievements.cpp

```diff
@@ -477,14 +572,27 @@ void Initialize() {
 
 	// Provide a logging function to simplify debugging
 	rc_client_enable_logging(g_rcClient, RC_CLIENT_LOG_LEVEL_VERBOSE, log_message_callback);
-
 	if (!System_GetPropertyBool(SYSPROP_SUPPORTS_HTTPS)) {
 		// Disable SSL if not supported by our platform implementation.
 		rc_client_set_host(g_rcClient, "http://retroachievements.org");
 	}
 
 	rc_client_set_event_handler(g_rcClient, event_handler_callback);
 
+#ifdef RC_CLIENT_SUPPORTS_RAINTEGRATION
+	if (g_Config.bAchievementsEnableRAIntegration) {
+		wchar_t szFilePath[MAX_PATH];
+		GetModuleFileNameW(NULL, szFilePath, MAX_PATH);
+		for (int64_t i = wcslen(szFilePath) - 1; i > 0; i--) {
+			if (szFilePath[i] == '\\') {
+				szFilePath[i] = '\0';
+			}
+		}
+		HWND hWnd = (HWND)System_GetPropertyInt(SYSPROP_MAIN_WINDOW_HANDLE);
+		rc_client_begin_load_raintegration(g_rcClient, szFilePath, hWnd, "PPSSPP", "1.0", &load_integration_callback, hWnd);
```

Comment:
"1.0" should be replaced with an actual version constant. The client name and version constant will be combined and used as the UserAgent for requests not being redirected through the HTTP handler (like editing code notes).